src/ja/mangamura/src/eu/kanade/tachiyomi/extension/ja/mangamura/MangaMuraFilters.kt

```diff
@@ -0,0 +1,110 @@
+package eu.kanade.tachiyomi.extension.ja.mangamura
+
+import eu.kanade.tachiyomi.source.model.Filter
+
+object Note : Filter.Header("NOTE: Ignored if using text search!")
+
+sealed class Select(
+    name: String,
+    val param: String,
+    values: Array<String>,
+) : Filter.Select<String>(name, values) {
+    open val selection: String
+        get() = if (state == 0) "" else state.toString()
+}
+
+class TypeFilter(
+    values: Array<String> = types.keys.toTypedArray(),
+) : Select("タイプ", "type", values) {
```

Comment:
You need to inherit from the classes defined the multisrc, or else they won't be applied